docs/source/user-guide/concepts/packages.rst

```diff
@@ -29,6 +29,29 @@ package. Directories are not included. Directories are created
 and removed as needed, but you cannot create an empty directory
 from the tar archive directly.
 
+.conda file format
+==================
+
+
+The .conda file format was introduced in conda 4.7 as a more
+compact, and thus faster, alternative to a tarball. 
+
+The .conda file format consists of an outer, uncompressed
+ZIP-format container, with two inner compressed .tar files.
```

Comment:
Could you mention that the inner ones are compressed with zstd or zstandard explicitly here @rrigdon?

Reply:
We should explain that the format inside doesn't matter, as long as libarchive know what to do with it. Zstd is the current implemention, but it is not part of the spec.